### DISCUSSION TITLE
How to enable scrollbar on hover the element and normally hides the scrollbar?

### DISCUSSION THREAD

**Discussion by montasim** [STRINGLABITSOLUTIONS/Bhojon#107] [Q&A] [answered]
I wanted to hide the scrollbar normally. But when hover the element I want to display the scrollbar.

**Comment by montasim** [comment]
Use overflow-scroll in tailwind css

**Comment by olx24** [comment, marked as answer]
Use overflow-scroll in tailwind css

**Comment by Preetiraj3697** [comment]
I have a sidebar that contains a scrollbar separately from the whole page scrollbar but it remain displayed. I want to show scrollbar only on hover. In the code below i have a class named "customizescrollbar" to customize the scroll bar and implement on the sidebar but it does not shows the scroll bar even on hover.

    .customizescrollbar::-webkit-scrollbar {
        display: none;
    }
    .customizescrollbar::-webkit-scrollbar: hover {
        width: 8px;
    }
    .customizescrollbar::-webkit-scrollbar-track {
        background: #f1f1f1; 
    }
    .customizescrollbar::-webkit-scrollbar-thumb {
        background: #CFCFCF; 
    }
